packages/lesswrong/components/ea-forum/EAHomeRightHandSide.tsx

```diff
@@ -596,45 +538,11 @@ export const EAHomeRightHandSide = ({classes}: {
   return <AnalyticsContext pageSectionContext="homeRhs">
     {!!currentUser && sidebarToggleNode}
     <div className={classes.root}>
-      {/* TODO: Remove after giving season ends */}
```

Comment:
Nice - I would also remove the link to the Giving Portal from the mobile menu (though nbd we leave that for a later PR)